src/transactions/api.ts

```diff
@@ -1,27 +1,51 @@
-import { createApi } from '@reduxjs/toolkit/query/react'
-import { baseQuery } from 'src/redux/api'
-import type { TokenTransaction } from 'src/transactions/types'
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react'
+import { type LocalCurrencyCode } from 'src/localCurrency/consts'
+import {
+  NetworkId,
+  TokenTransactionTypeV2,
+  type PageInfo,
+  type TokenTransaction,
+} from 'src/transactions/types'
+import networkConfig from 'src/web3/networkConfig'
 
-export const FIRST_PAGE_TIMESTAMP = 0 // placeholder
+export const FIRST_PAGE_CURSOR = 'FIRST_PAGE'
 
 export type TransactionFeedV2Response = {
   transactions: TokenTransaction[]
-  pageInfo: {
-    hasNextPage: boolean
-  }
+  pageInfo: PageInfo
 }
 
+const ALLOWED_ZERION_NETWORKS: NetworkId[] = [
+  NetworkId['ethereum-mainnet'],
+  NetworkId['celo-mainnet'],
+  NetworkId['op-mainnet'],
+  NetworkId['base-mainnet'],
+  NetworkId['arbitrum-one'],
+  NetworkId['polygon-pos-mainnet'],
+]
+
+const baseQuery = fetchBaseQuery({
+  baseUrl: networkConfig.getWalletTransactionsUrl,
+  headers: { Accept: 'application/json' },
+})
+
 export const transactionFeedV2Api = createApi({
   reducerPath: 'transactionFeedV2Api',
   baseQuery,
   endpoints: (builder) => ({
     transactionFeedV2: builder.query<
       TransactionFeedV2Response,
-      { address: string; endCursor: number }
+      {
+        address: string
+        localCurrencyCode: LocalCurrencyCode
+        endCursor: PageInfo['endCursor'] | undefined
+      }
     >({
-      query: ({ address, endCursor }) => {
-        const cursor = endCursor ? `?endCursor=${endCursor}` : ''
-        return `/wallet/${address}/transactions${cursor}`
+      query: ({ address, localCurrencyCode, endCursor }) => {
+        const networkIds = ALLOWED_ZERION_NETWORKS.join('&networkIds[]=')
+        const includeTypes = Object.values(TokenTransactionTypeV2).join('&includeTypes[]=')
```

Comment:
These are currently required by the API. Should it be managed by the API itself maybe? Unless there's a certain need to set this via dynamic configs - I don't see why this can't be omitted from the client side.

Reply:
@kathaypacific I only meant the highlighted lines. Currently, API requires to provide `networkIds` and `includeTypes` in the query. But this can be automatically appended on the blockhain-api side, without explicitly setting it from the client.

Reply:
ah, i see! blockchain-api needs to serve multiple versions of clients, it's reasonable to assume that clients might expect different values for these. e.g. if we roll out a new chain, we may need to make some code changes to make it work in certain scenarios so the feed should only return transactions that the client can handle. you can argue that the server can read the client version and decide on the parameters, i'm not really sure there is an advantage to that though

Reply:
@kathaypacific Gotcha! Should I then set it up in a way where both `networkIds` and `includeTypes` are defined in `.env` or should it be dynamic config?

Reply:
i was wondering about dynamic config, but i think that's unnecessary complexity. one thing that we've tried to do is to ensure that adding new chains is easy (it is one of the PR description prompts) so in that line of thinking, we just want to do this in a way that will minimise chances of forgetting to update this. so my suggestion is to use existing config wherever possible rather than declaring new ones (e.g. use `Object.values(networkConfig.networkToNetworkId)`)